addons/arsenal/functions/fnc_onArsenalClose.sqf

```diff
@@ -69,6 +69,8 @@ camDestroy GVAR(camera);
 if (!isNil QGVAR(moduleUsed)) then {
     GVAR(moduleUsed) = nil;
     objNull remoteControl GVAR(center);
+    player switchCamera cameraView;
+    GVAR(center) setVariable ["bis_fnc_moduleRemoteControl_owner", nil, true];
```

Comment:
Unnecessary. This variable is never set when the arsenal is opened in `moduleUsed` mode.